### PR TITLE
fix: onboarding template fixes

### PR DESCRIPTION
**Pull request by raquelmsmith**
## Problem

A few problems I saw people run into on launch yesterday:

- The iframe would get ridiculously long if the vars were long. Constrain to 90vh max
- The slider buttons in the toolbar didn't have any bg color (because colors aren't reliably available in the toolbar and I can't figure out why)
- This simply does not work on small screens. That can be added later, but for now only include it on desktop or tablet screens
- Events take a while to be queryable. Added the live events list to generate a list of hosts that might also be useful for this.

## Changes

Explained above.

👉 _Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review._

## How did you test this code?

👀

### CODE REVIEW

File: frontend/src/toolbar/actions/ActionAttribute.tsx [hunk 1]
```diff
@@ -58,15 +58,19 @@ export function ActionAttribute({
...
-                <LemonSwitch
-                    size="small"
-                    checked={automaticCreationIncludedPropertyKeys.includes(attribute)}
-                    onChange={(checked) =>
-                        checked
-                            ? addAutomaticCreationIncludedPropertyKey(attribute)
-                            : removeAutomaticCreationIncludedPropertyKey(attribute)
-                    }
-                />
+                <>
```
Comment (zlwaterfield) [inline frontend/src/toolbar/actions/ActionAttribute.tsx:61]: Nite: this fragment wrapper seems unneeded

File: frontend/src/scenes/onboarding/sdks/sdksLogic.tsx [hunk 2]
```diff
@@ -118,6 +119,20 @@ export const sdksLogic = kea<sdksLogicType>([
...
             },
         ],
+        combinedSnippetAndLiveEventsHosts: [
+            (selectors) => [selectors.snippetHosts, selectors.eventHosts],
+            (snippetHosts: string[], eventHosts: string[]): string[] => {
+                const combinedSnippetAndLiveEventsHosts = snippetHosts
+                for (const host of eventHosts) {
+                    const hostProtocol = new URL(host).protocol
+                    const currentProtocol = window.location.protocol
+                    if (hostProtocol === currentProtocol && !combinedSnippetAndLiveEventsHosts.includes(host)) {
```
Comment (zlwaterfield) [inline frontend/src/scenes/onboarding/sdks/sdksLogic.tsx:129]: Idea: you could use a `Set` here instead which would make the logic a little cleaner.

File: frontend/src/scenes/onboarding/productAnalyticsSteps/DashboardTemplateConfigureStep.tsx [hunk 3]
```diff
@@ -243,11 +251,11 @@ export const OnboardingDashboardTemplateConfigureStep = ({
                     </div>
                 ) : (
                     <div className="grid grid-cols-6 space-x-6 min-h-[80vh]">
-                        <div className="col-span-4 relative">
+                        <div className="col-span-4 relative max-h-[100vh] overflow-y-hidden">
                             {browserUrl && iframeBanner?.level != 'error' ? (
                                 <div className="border border-1 border-border-bold rounded h-full w-full flex flex-col">
                                     <UrlInput iframeRef={iframeRef} />
-                                    <div className="m-2 grow rounded">
+                                    <div className="m-2 grow rounded ">
```
Comment (zlwaterfield) [inline frontend/src/scenes/onboarding/productAnalyticsSteps/DashboardTemplateConfigureStep.tsx:258]:
```suggestion
                                    <div className="m-2 grow rounded">
```

File: frontend/src/scenes/onboarding/productAnalyticsSteps/DashboardTemplateConfigureStep.tsx [hunk 4]
```diff
@@ -122,15 +124,21 @@ export const SiteChooser = (): JSX.Element => {
...
                                 the HTML snippet wherever you want to track events, then come back here.
                             </p>
+                            {isCloud && (
+                                <p className="text-muted italic">
+                                    Note: Sites must be served over HTTPS to be selected.
+                                </p>
+                            )}
                             <div className="space-y-2">
-                                {snippetHosts.map((host) => (
+                                {combinedSnippetAndLiveEventsHosts.concat('https://posthog.com').map((host) => (
```
Comment (zlwaterfield) [inline frontend/src/scenes/onboarding/productAnalyticsSteps/DashboardTemplateConfigureStep.tsx:141]: I'm not sure I understand why you added this.

**Review by zlwaterfield — APPROVED** [review]
Sweet, looks good!

File: frontend/src/scenes/onboarding/productAnalyticsSteps/DashboardTemplateConfigureStep.tsx (hunk 4 above)
Comment (raquelmsmith) [reply frontend/src/scenes/onboarding/productAnalyticsSteps/DashboardTemplateConfigureStep.tsx:141]: Ha oops that wasn't supposed to be there

File: frontend/src/scenes/onboarding/sdks/sdksLogic.tsx (hunk 2 above)
Comment (raquelmsmith) [reply frontend/src/scenes/onboarding/sdks/sdksLogic.tsx:129]: My editor complained about the Set so I did it this way

**Pull request merged**